### PR TITLE
evil-winrm: init at 3.3

### PR DESCRIPTION
**Pull request by elohmeier**
###### Motivation for this change
#81418

###### Things done
Added package evil-winrm.

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/evil-winrm/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  installPhase = ''
+    mkdir -p $out/bin
+    cp evil-winrm.rb $out/bin/evil-winrm
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/Hackplayers/evil-winrm";
+    changelog = "https://github.com/Hackplayers/evil-winrm/releases/tag/v${version}";
+    description = "WinRM shell for hacking/pentesting";
+    license = licenses.lgpl3;
```
Comment (fabaff) [inline pkgs/tools/security/evil-winrm/default.nix:34]: According to a section in the [README](https://github.com/Hackplayers/evil-winrm#disclaimer--license) is the license `LGPLv3+`.

```suggestion
    license = licenses.lgpl3Plus;
```

**Review by fabaff — COMMENTED** [review]
Rest looks solid to me.

File: pkgs/tools/security/evil-winrm/default.nix (hunk 1 above)
Comment (elohmeier) [reply pkgs/tools/security/evil-winrm/default.nix]: Thanks, fixed that & rebased onto master.

File: pkgs/tools/security/evil-winrm/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  pname = "evil-winrm";
+  version = "3.3";
+
+  src = fetchFromGitHub
+    {
+      owner = "Hackplayers";
+      repo = "evil-winrm";
+      rev = "v${version}";
+      sha256 = "sha256-uWhRkq7I/XRWSUpR8lWRhDImE6x0pX9/B3gKhRIhkf8=";
+    };
```
Comment (SuperSandro2000) [inline pkgs/tools/security/evil-winrm/default.nix:13]:
```suggestion
  src = fetchFromGitHub {
    owner = "Hackplayers";
    repo = "evil-winrm";
    rev = "v${version}";
    sha256 = "sha256-uWhRkq7I/XRWSUpR8lWRhDImE6x0pX9/B3gKhRIhkf8=";
  };
```

**Pull request merged**